lib/transactions/dapp.js

```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating dapps in Lisk Blockchain
```

Comment:
Dapps module provides functions used to create dapps (blockchain apps) for the Lisk Blockchain